src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -37,6 +37,33 @@ public void BuildBasicApplicationRelease ()
 			}
 		}
 
+		[Test]
+		public void BuildBasicApplicationReleaseProfiledAot ()
+		{
+			var proj = new XamarinAndroidApplicationProject () {
+				IsRelease = true,
+				AndroidEnableProfiledAot = true,
+			};
+			using (var b = CreateApkBuilder ("temp/BuildBasicApplicationRelease")) {
+				Assert.IsTrue (b.Build (proj), "Build should have succeeded.");
+				StringAssertEx.ContainsRegex (@"\[aot-compiler stdout\] Using profile data file.*lib/xamarin.android/xbuild/Xamarin/Android/startup\.aotprofile", b.LastBuildOutput, "Should use default AOT profile");
+			}
+		}
+
+		[Test]
+		public void BuildBasicApplicationReleaseProfiledAotWithoutDefaultProfile ()
+		{
+			var proj = new XamarinAndroidApplicationProject () {
+				IsRelease = true,
+				AndroidEnableProfiledAot = true,
+			};
+			proj.SetProperty (proj.ActiveConfigurationProperties, "AndroidUseDefaultAotProfile", "false");
+			using (var b = CreateApkBuilder ("temp/BuildBasicApplicationRelease")) {
```

Comment:
Can you replace this with `Path.Combine ("temp", TestName);` other wise the two tests will collide with each other.